src/providers/pkcs11/utils.rs

```diff
@@ -119,3 +126,25 @@ pub fn digest_info(alg: AsymmetricSignature, hash: Vec<u8>) -> Result<Vec<u8>> {
     // should not fail - if it does, there's some error in our stack
     .map_err(|_| ResponseStatus::PsaErrorGenericError)
 }
+
+pub fn ec_params(ecc_family: EccFamily, bits: usize) -> Result<ECParameters> {
+    Ok(ECParameters::NamedCurve(match (ecc_family, bits) {
+        // The following "unwrap()" should be ok, as they cover constant conversions
+        (EccFamily::SecpR1, 192) => {
+            ObjectIdentifierAsn1(String::from("1.2.840.10045.3.1.1").try_into().unwrap())
```

Comment:
Isn't there in some of the picky crates, those defined as constants or methods?

Reply:
There is - in the ones we can't use because of compiler limitations 😬 I added them recently, so they're in the most recent version only.

Reply:
ah right, that was it! No worries then.